components/listingsPage/filters.js

```diff
@@ -166,226 +175,176 @@ const Filters = () => {
               textDecoration: "none",
             }}
           >
-            Clear
+            {t("headings.clear")}
           </a>
         )}
       </Heading>
       <FormDiv>
         <FilterFormFIeld>
           <Legend>
-            Age at time of travel
+            {t("headings.age_at_time_of_travel")}
             <Tooltip hover text="To be supplied" />
           </Legend>
           <Select name="age" value={age.age} onChange={(e) => handleAge(e)}>
-            <option value="">Age at time of travel</option>
+            <option value="">{t("headings.age_at_time_of_travel")}</option>
             {ageRange()}
           </Select>
         </FilterFormFIeld>
         <FilterFormFIeld>
           <Legend>
-            Filter by insurance type
+            {t("headings.filter_by_insurance_type")}
             <Tooltip minWidth="300px" hover text={insuranceTypeTip} />
           </Legend>
-          <Radio
-            value="Single Trip"
-            checked={insuranceType.insuranceType === "Single Trip"}
-            onChange={(e) => handleInsuranceType(e)}
-            label="Single Trip"
-            name="insuranceType"
-          />
-          <Radio
-            value="Annual Trip"
-            checked={insuranceType.insuranceType === "Annual Trip"}
-            onChange={(e) => handleInsuranceType(e)}
-            label="Annual Trip"
-            name="insuranceType"
-          />
+          {t("filters.insuranceType", { returnObjects: true }).map(
+            ({ type, value }, i) => (
+              <Radio
+                key={i}
+                checked={insuranceType.insuranceType === value}
+                onChange={(e) => handleInsuranceType(e)}
+                label={type}
+                name="insuranceType"
+                id={`type-${i}`}
```

Comment:
I'm not sure if the `id` attribute is required for logic outside of the form semantics but if not it isn't required here (and in other suchlike components)

Reply:
There are some input manipulation that might require it going forward

Reply:
That's fine! Cheers 👍